app/org/sagebionetworks/bridge/services/StudyService.java

```diff
@@ -586,6 +599,9 @@ private void sanitizeHTML(Study study) {
         
         template = study.getResetPasswordTemplate();
         study.setResetPasswordTemplate(sanitizeEmailTemplate(template));
+        
+        template = study.getEmailSignInTemplate();
+        study.setEmailSignInTemplate(sanitizeEmailTemplate(template));
```

Comment:
It looks like we don't have any tests for sanitizing email templates. Is this something we should be testing?